### blibliotecas iniciais 

In [12]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import warnings
import os
warnings.filterwarnings("ignore")


### Blibliotecas para tratamento de imagem

In [13]:
from skimage.color import rgb2gray
from skimage.filters import try_all_threshold, threshold_otsu, threshold_local, gaussian
from skimage.transform import rotate, rescale
from skimage import morphology
import cv2

In [14]:
# G:\Meu Drive\Renan-Lemes-TCC\dataset\Potato\train\Potato Early blight
# G:\Meu Drive\Renan-Lemes-TCC\dataset\Potato\train\Potato Healthy
# G:\Meu Drive\Renan-Lemes-TCC\dataset\Potato\train\Potato Late blight

In [18]:
trainEarly = os.listdir('G:\\Meu Drive\Renan-Lemes-TCC\\dataset\\Potato\\train\\Potato Early blight')
trainHeal = os.listdir('G:\\Meu Drive\\Renan-Lemes-TCC\\dataset\\Potato\\train\\Potato Healthy')
trainLate = os.listdir('G:\\Meu Drive\\Renan-Lemes-TCC\\dataset\\Potato\\train\\Potato Late blight')


In [20]:
print('-----train-----')
print(len(trainHeal))
print(len(trainEarly))
print(len(trainLate))

-----train-----
2432
3532
3521


### Ai decidi Pegar apenas  as 2432 das saudaveis, e metade das Late e metade Early ou seja 1216 cada.

In [29]:
print(round(len(trainHeal)/2 ))

1216
